torch/cuda/profiler.py

```diff
@@ -45,6 +45,13 @@ def init(output_file, flags=None, output_mode="key_value"):
 
 
 def start():
+    """
+    Starts Cuda profile data collection
+
+    Example:
+        >>> profiler = Profiler();
+        >>> profiler.start()
```

Comment:
I think the usage would be more like `torch.cuda.profiler.start()`, I don't think there's a `Profiler()` object that would have to be constructed first